src/widgets/JudgePanel.ts

```diff
@@ -276,6 +278,12 @@ export class JudgePanel extends BoxPanel {
       ) {
         // User canceled the execution. Do nothing.
       } else {
+        this._executed.emit({
+          widget: this,
+          cell: this.model.codeModel as ICodeCellModel,
+          sucess: false,
+          error: e
```

Comment:
You might want the error for the execution itself. This error is not a Python error. Check reply variable.

Reply:
I wanted to contain an error when execution failed, including a kernel error. (I referred to JuptyterLab internal code. https://github.com/jupyterlab/jupyterlab/blob/3.6.x/packages/notebook/src/actions.tsx#L2204 )